jobspec/parse_task.go

```diff
@@ -263,6 +263,7 @@ func parseTask(item *ast.ObjectItem, keys []string) (*api.Task, error) {
 			"max_files",
 			"max_file_size",
 			"enabled",
```

Comment:
```suggestion
			"enabled", COMPAT(1.7.0): remove in favor of disabled
```

or 1.6 if we go with that